pkg/router/f5/f5.go

```diff
@@ -1823,3 +1865,9 @@ func (f5 *f5LTM) deleteCertParts(routename string,
 
 	return nil
 }
+
+// escapeiControlUriPath returns a fully qualified `escaped` URI path that
+// can be used in the iControl REST calls.
+func escapeiControlUriPath(pathName string) string {
+	return strings.Replace(pathName, "/", "~", -1)
+}
```

Comment:
I think this was first call added - squashed a few commits into 1. Will move this to the top or move those below I think - which might be better.